### PR TITLE
test(cli): add unit tests

### PR DESCRIPTION
**Pull request by eunjae-lee**
## What?

This PR adds unit tests for `main.ts` and move some codes around to make it easier.

You can read more about this at https://storyblok.atlassian.net/browse/EXT-1118?focusedCommentId=48609

## Why?

JIRA: EXT-1118

## How to test? (optional)

```sh
yarn test:cli
```

### CODE REVIEW

File: packages/cli/src/commands/create/index.ts [hunk 1]
```diff
@@ -1,45 +1,15 @@
...
   selectPackageManager,
+  selectRepositoryStructure,
+  isValidStructure,
 } from '../../utils'
 import { CreateFunc, CreateArgs } from './types'
-import { PackageManager } from '../types'
 
 export type { CreateArgs }
 
-const selectRepositoryStructure = async () => {
```
Comment (eunjae-lee) [inline packages/cli/src/commands/create/index.ts:14]: This has been moved to `utils` with no change. I've moved it to easily mock it later.

File: packages/cli/src/commands/add.ts [hunk 2]
```diff
@@ -21,11 +21,7 @@ import {
   runCommand,
   selectPackageManager,
 } from '../utils'
-import type { PackageManager } from './types'
-
-export type Template = 'vue2'
-
-export type Structure = 'standalone' | 'monorepo'
```
Comment (eunjae-lee) [inline packages/cli/src/commands/add.ts:28]: moving these types to `types.ts` file.

**Comment by eunjae-lee** [comment]
Current dependencies on/for this PR:
* main
  * **PR #238** <a href="https://app.graphite.dev/github/pr/storyblok/field-plugin/238" target="_blank">Graphite</a>  👈
    * **PR #239** <a href="https://app.graphite.dev/github/pr/storyblok/field-plugin/239" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/storyblok/field-plugin/238?utm_source=stack-comment).

**Review by BibiSebi — APPROVED** [review]
💯

**Comment by eunjae-lee** [comment]
@eunjae-lee started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/storyblok/field-plugin/238).

**Comment by eunjae-lee** [comment]
[Graphite](https://app.graphite.dev/github/pr/storyblok/field-plugin/238) couldn't merge this PR because it had conflicts with the trunk branch.

**Comment by eunjae-lee** [comment]
@eunjae-lee merged this pull request with [Graphite](https://app.graphite.dev/github/pr/storyblok/field-plugin/238).

**Pull request merged**